src/main/java/dev/joopie/jambot/repository/PlayHistoryRepository.java

```diff
@@ -1,10 +1,19 @@
 package dev.joopie.jambot.repository;
 
 import dev.joopie.jambot.model.PlayHistory;
+import jakarta.transaction.Transactional;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.ListCrudRepository;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface PlayHistoryRepository extends ListCrudRepository<PlayHistory, Long> {
     void deleteByGuildId(String guildId);
+
+    @Modifying
+    @Transactional
+    @Query(value = "DELETE FROM play_history WHERE id = (SELECT id FROM play_history WHERE user_id = :userId AND track_id = :trackId AND DATE(created_at) = CURRENT_DATE ORDER BY created_at DESC LIMIT 1)", nativeQuery = true)
```

Comment:
```suggestion
    @Query(value = "DELETE FROM play_history WHERE user_id = :userId AND track_id = :trackId ORDER BY created_at DESC LIMIT 1)", nativeQuery = true)
```

Werkt dit niet gewoon? 
Hernoem de methode naar iets met `delete latest by user id and track id` oid.